lighthouse-core/gather/connections/cri.js

```diff
@@ -108,10 +115,11 @@ class CriConnection extends Connection {
       log.warn('CriConnection', 'disconnect() was called without an established connection.');
       return Promise.resolve();
     }
-    this._ws.removeAllListeners();
-    this._ws.close();
-    this._ws = null;
-    return Promise.resolve();
+    return this._runJsonCommand(`close/${this._pageId}`).then(_ => {
+      this._ws.removeAllListeners();
+      this._ws.close();
+      this._ws = null;
```

Comment:
I don't know when we'd ever reuse a cri connection instance, but since we null out `_ws`, null out `_pageId` as well?